include/mbedtls/config_psa.h

```diff
@@ -1024,6 +1024,58 @@ extern "C" {
 #define PSA_WANT_ALG_SOME_PAKE 1
 #endif
 
+/* Backward compatibility */
```

Comment:
I'd go further: "Temporary internal migration helpers". We don't want users to think they can use those!